providers/constants/oracle.go

```diff
@@ -0,0 +1,58 @@
+package constants
+
+import (
+	"math/big"
+
+	apihandlers "github.com/skip-mev/slinky/providers/base/api/handlers"
+	mmtypes "github.com/skip-mev/slinky/x/marketmap/types"
+)
+
+type (
+	// PriceAPIQueryHandler is a type alias for the API query handler that is used to fetch price data
+	// from the price providers.
+	PriceAPIQueryHandler = apihandlers.APIQueryHandler[mmtypes.Ticker, *big.Int]
+
+	// PriceAPIDataHandler is a type alias for the API data handler that is used to fetch resolve http
+	// requests and parse the response into price data.
+	PriceAPIDataHandler = apihandlers.APIDataHandler[mmtypes.Ticker, *big.Int]
+)
+
+var (
+	// The following are the default tickers for the oracle sidecar.
+	//
+	// USD denominated tickers.
+	ATOM_USD     = mmtypes.NewTicker("ATOM", "USD", 8, 1)
+	AVAX_USD     = mmtypes.NewTicker("AVAX", "USD", 8, 1)
```

Comment:
Want to suppress this warning from the linter?  I like having constants all caps like this.  We could exclude this rule from the golangci config